bios/maddy.md

```diff
@@ -1,13 +1,13 @@
-# Jen Reiher
-![Picture of Jen](https://www.canadalearningcode.ca/wp-content/uploads/2017/07/jen.jpg)
+# Maddy Heidt
+![Picture of Maddy](https://www.facebook.com/photo.php?fbid=10202606383516169&l=3b6e3fe5c5)
```

Comment:
it looks like this link might not work - but let's give it a try.
if not, you might just need to right click on the image and "copy link address" out of facebook